addons/openvpn/networkpolicy.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 The Kubermatic Kubernetes Platform contributors.
```

Comment:
we would need a similar policy for konnectivity-agent as well

Reply:
Yes, I was not sure if I can test it already as there is ongoing development. What do you think?

Reply:
Some tweaks still needed, but it is already working in kubermatic dev environment.